deepcell_tracking/isbi_utils.py

```diff
@@ -352,23 +370,37 @@ def benchmark_division_performance(trk_gt, trk_res, path_gt, path_res):
     trks = load_trks(trk_res)
     lineage_res, _, y_res = trks['lineages'][0], trks['X'], trks['y']
 
-    # Produce ISBI style text doc to work with
-    trk_to_isbi(lineage_gt, path_gt)
-    trk_to_isbi(lineage_res, path_res)
+    # Produce ISBI style array to work with
+    if path_gt or path_res is not None:
+        print("UserWarning: The 'path_gt` and 'path_res' arguments are deprecated.")
+        trk_to_isbi(lineage_gt, path_gt)
+        trk_to_isbi(lineage_res, path_res)
+    else:
+        gt = trk_to_isbi(lineage_gt)
+        res = trk_to_isbi(lineage_res)
```

Comment:
This loop doesn't matter - if path is `None` then it will work as expected anyway.
I think the warning is all that needs to be in the conditional.